src/task/seventh/Cupboard.java

```diff
@@ -0,0 +1,36 @@
+package task.seventh;
+
+public class Cupboard<K, V> {
+    private K key;
```

Comment:
where will you save your data?
add arrays or list for keys and values

Reply:
What are you mean? must I use map?